cmp/grid/Grid.js

```diff
@@ -597,6 +596,14 @@ export class Grid extends Component {
         return column.isTreeColumn ? node.data[column.field] : value;
     }
 
+    onKeyDown = (evt) => {
+        if (evt.ctrlKey && evt.key == 'a') {
```

Comment:
Should we add `this.model.selModel.mode === 'multiple'` to this conditional? Don't think we want Ctrl + a to do anything for a single-mode selection model